scheduler/src/main/java/org/apache/mesos/elasticsearch/scheduler/state/StatePath.java

```diff
@@ -43,4 +43,9 @@ public Boolean exists(String key) throws IOException {
         }
         return exists;
     }
+
+    public void rm(String key) throws IOException {
+        key = key.replace(" ", "");
```

Comment:
Where would these " " come from? Without explanation it looks like a manipulation of a key to remove